app/views/coronavirus_landing_page/components/landing_page/_announcements_section.html.erb

```diff
@@ -1,29 +0,0 @@
-<div class="govuk-grid-row">
-  <div class="govuk-grid-column-one-third">
-    <%= render "govuk_publishing_components/components/heading", {
-      text: details.announcements_label,
-      margin_bottom: 6,
-      font_size: "m"
-    } %>
-  </div>
-
-  <div class="govuk-grid-column-two-thirds">
-    <%= render partial: 'coronavirus_landing_page/components/shared/announcements', locals: { announcements: details.announcements } %>
-
-    <% if details.see_all_announcements_link.present? %>
-      <p class="govuk-body">
-        <a
-          href="<%= details.see_all_announcements_link["href"] %>"
-          class="govuk-link"
-          data-module="gem-track-click"
-          data-track-category="pageElementInteraction"
-          data-track-action="Announcements"
-          data-track-label="<%= details.see_all_announcements_link["href"] %>"
-        >
-          <%= details.see_all_announcements_link["text"] %>
-        </a>
-      </p>
-    <% end %>
-    <%= render partial: 'coronavirus_landing_page/components/landing_page/risk_level_section', locals: { data: details.risk_level } if details.risk_level %>
```

Comment:
I think this partial, `_risk_level_section.html.erb` is only rendered here, so should be removed as well.

Reply:
Yeah - quire right - removed